app/routes/report/tmburMutationBurden.js

```diff
@@ -26,7 +26,7 @@ router.use('/', async (req, res, next) => {
       where: {reportId: req.report.id},
     });
   } catch (error) {
-    logger.error(`Unable to query Tmbur Mutation Burden for report ${req.report.ident} error: ${error}`);
+    logger.error(`Unable to query Tmbur Mutation Burden ${req.report.ident} error: ${error}`);
```

Comment:
might be better to leave in 'for report' otherwise it seems like the ident is for the tmbur mutation burden in this msg